src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -139,8 +139,10 @@ public function databaseDownload($siteName = 'default')
      *   Path to the AWS configuration directory.
      * @param string $awsConfigFilePath
      *   Path to the AWS configuration file.
+     *
+     * @return \Robo\Result
      */
-    protected function configureAwsCredentials(string $awsConfigDirPath, string $awsConfigFilePath)
+    protected function configureAwsCredentials(string $awsConfigDirPath, string $awsConfigFilePath): Result
```

Comment:
Question: Why does the docblock need fully qualified class (\Robo\Result) wherease in the function signature the returned class is just Result instead of \Robo\Result. I assume this file has a `use \Robo\Result` or such, which is why the codebase resolves the return type. Is doc syntax not able to resolve the intended class the way the code does?

Reply:
> Why does the docblock need fully qualified class (\Robo\Result) wherease in the function signature the returned class is just Result instead of \Robo\Result.

I've wondered this too when I first started getting my hands dirty with OO and D8. FWIW this is just how I've always seen it done and what documentation always says to do. 

> Is doc syntax not able to resolve the intended class the way the code does?

That is my best guess, but I don't have a good answer either.